MaaSSim/decisions.py

```diff
@@ -226,7 +226,7 @@ def f_match(**kwargs):
                          'request': pax_request,
                          'wait_time': mintime,
                          'travel_time': ttrav,
-                         'fare': platform.platform.fare * sim.pax[i].request.dist / 1000}  # make an offer
+                         'fare': max(platform.platform.base_fare + platform.platform.km_fare * sim.pax[i].request.dist / 1000, platform.platform.min_fare)}  # make an offer
```

Comment:
let's do like this:
`max(platform.platform.get('base_fare',0)+platform.platform.fare*sim.pax[i].request.dist / 1000, platform.platform.get(min_fare,0))`

this way you may add your fields and if they are not present the default values are taken

Reply:
Thanks, I'll incorporate this